src/graphorm/index.ts

```diff
@@ -247,10 +255,24 @@ const obj = new GraphORM({
     },
   },
   Post: {
-    additionalQuery: (ctx, alias, qb) =>
-      qb
+    additionalQuery: (ctx, alias, qb) => {
+      qb = qb
         .andWhere(`"${alias}"."deleted" = false`)
-        .andWhere(`"${alias}"."visible" = true`),
+        .andWhere(`"${alias}"."visible" = true`);
+
+      if (ctx.isPlus) {
+        qb = qb
+          .innerJoin(Settings, 's', 's."userId" = :userId', {
+            userId: ctx.userId,
+          })
+          .addSelect(
+            `"s"."flags"->'clickbaitShieldEnabled'`,
+            'clickbaitShieldEnabled',
+          );
+      }
+
+      return qb;
+    },
```

Comment:
FML tried for so long to figure out a way to select user settings from `createSmartTitleField` function... if only it had accepted async functions.